app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from __future__ import annotations
+
+from app.copy_content import manager_call
+
+
+def move_file(command: str) -> None:
+    split_cmd = command.split()
+    if len(split_cmd) != 3 or split_cmd[0] != "mv":
```

Comment:
The condition checks if the command is in the correct format. Ensure that the command strictly follows the format 'mv <source_file> <moved_file>'. Consider adding more detailed error messages or handling for different types of incorrect input.